src/getting-started.md

```diff
@@ -33,71 +33,65 @@ instead.
 
 ### Cards
 
-A question and answer pair is called a 'card'. This is based on a paper
-flashcard with a question on one side and the answer on the back. In
-Anki a card doesn’t actually look like a physical card, and when you
-show the answer the question remains visible by default. For example, if
-you’re studying basic chemistry, you might see a question like:
+A question and answer pair is called a _card_. It's similar to a paper
+flashcard with a question on front and answer on the back. However, in
+Anki, a card doesn't actually look like a physical card, and when you
+look at the answer the question remains visible by default. For example, if
+you're studying basic chemistry, you might see a question like:
 
     Q: Chemical symbol for oxygen?
 
-After thinking about it, and deciding the answer is O, you click the
-show answer button, and Anki shows you:
+After deciding the answer is O, you click the
+"Show Answer" button, and Anki shows you:
 
     Q: Chemical symbol for oxygen?
     A: O
 
-After confirming that you are correct, you can tell Anki how well you
-remembered, and Anki will choose a next time to show you again.
+After confirming that you are correct, you tell Anki how well you
+remembered the answer, and Anki will choose when to show you the card again. If, for example, Anki decides to show you the card again in 30 days, we say the card has a 30 day interval.
```

Comment:
I'm not sure how to do this properly. The manual doesn't explicitly explain intervals anywhere but we need it here as defining young and mature later needs understanding interval.

"For example" isn't a good way to transition to this, which is why I started with a "If". I agree with 30d versus 3d but would get rid of the hyphen.

Reply:
Oh yes, I see what you mean about needing to transition it better, but I think the multiple commas makes the sentence stutter getting off the ground. 

If you want to define it here -- how about doing it without an example? 
```suggestion
remembered the answer, and Anki will choose when to show you the card again. The time between now and when Anki schedules the card for you to study it again is called the card's _interval_.
```

Reply:
I think with example is better; the definition isn't that easy for me. This might not be that bad though:

```suggestion
remembered the answer, and Anki will choose when to show you the card again. For example, Anki might decide to show you the card again in 3 days. In this case, we say the card now has a 3 day interval.
```

Reply:
That works. [And I know you don't like the hyphen, but it's a compound adjective before a noun, so [that's the rule](https://www.grammarly.com/blog/compound-adjectives/).]